stencils/UninitializedSubject.stencil

```diff
@@ -14,7 +14,7 @@ class UninitializedSubject<Input: KotlinObject, Output: Any>: ObservableObject {
     @Published var value: Output?
 
     private var input: Input?
-    private let disposeBag = DisposeBag(allowFreezing: false)
```

Comment:
Maybe this also can be part of configuration? So we can tune it

Reply:
Allow freezing is removed so there is nothing to tune

Reply:
Or do you mean, pass a custom disposeBag?

Reply:
I forgot that fact it was removed, so never mind